### ISSUE TITLE
LongVideo-Reason Dataset

### ISSUE THREAD

**Issue by XiaoYee** [NVlabs/Long-RL#1]
Thanks for your great work. Will you release the LongVideo-Reason Dataset?

**Comment by JoshonSmith** [comment]
> You can search for `LongVideo-Reason` on hf.

The `LongVideo-Reason`  dataset needs `longvila_videos`.
Where can we download `longvila_videos`?

**Comment by yukang2017** [comment]
For test videos, I include it in the `LongVideo-Reason`.
For training videos, the name of `videos` is the YouTube ID of it, you can download them on YouTube.

**Comment by yukang2017** [comment]
@XiaoYee @JoshonSmith You can download all these tar.gz files, and extract them into a directory named `LongVideo-Reason`
https://huggingface.co/datasets/Perflow-Shuai/movie_mp4/tree/main

**Comment by sming256** [comment]
Thank you for the great work!

Could you please clarify which parts of the videos in the released JSON are used for SFT and which parts are used for RL?

**Comment by yukang2017** [comment]
Hi @sming256 , thanks for your question. We use 1/3 in the train set for SFT and 2/3 for RL. But I am not sure this is the optimal setting for various models. You could have a try.

**Comment by yukang2017** [comment]
Hi @XiaoYee , I used 128 A100 GPUs for SFT training within 1 day for SFT and 128 A100 GPUs for RL training on 2-3 days.

Hi @sming256 , yes.

**Comment by yukang2017** [comment]
Hi, in `LongVideo-Reason/longvideo-reason`, validation set is used for the validation during training, test is used for the final evaluation. Thanks.

**Comment by mwxely** [comment]
> Hi, in `LongVideo-Reason/longvideo-reason`, validation set is used for the validation during training, test is used for the final evaluation. Thanks.

Thanks for your response! I will refer to https://huggingface.co/datasets/LongVideo-Reason/longvideo-reason/tree/main →val.jsonl

**Comment by vvukimy** [comment]
Great Work!!! Thanks for releasing the dataset.

Could you please provide the `problem_type` labels for the training set? I noticed that in the dataset, only `test.jsonl` has detailed `problem_type` annotations, while the rest are all marked as "general".

**Comment by sinbycos24** [comment]
Hi, sorry I am new. How do I finetune this model with my own created datasets?
